tests/js/web/DropZoneComponentUtilsTest.js

```diff
@@ -0,0 +1,217 @@
+import DropZoneComponentUtils from '../../../js/DropZone/DropZoneComponentUtils';
+import MimeTyper from '../../../js/libs/MimeTyper';
+
+describe('DropZoneComponentUtils', () => {
+    let utils;
+    let mimeStub;
+
+    beforeEach(() => {
+        mimeStub = sinon.createStubInstance(MimeTyper);
+        mimeStub.getIconClass.returns('foo');
+        utils = new DropZoneComponentUtils(mimeStub);
+    });
+
+    describe('getOptionsFromAttrs()', () => {
+        it('should parse data-dropzone attributes', () => {
+            const $node = $('<div data-dropzone-foo="bar" data-foo="bar"></div>');
+
+            expect(utils.getOptionsFromAttrs($node[0]))
+                .to.deep.equal({ foo: 'bar' });
+        });
+
+        it('should covert the whitelist option to an array', () => {
+            const $node = $('<div data-dropzone-whitelist="image/* jpeg"></div>');
+
+            expect(utils.getOptionsFromAttrs($node[0]))
+                .to.deep.equal({ whitelist: ['image/*', 'jpeg'] });
+        });
+
+        it('should parse bools', () => {
+            const $node = $('<div data-dropzone-foo="true" data-dropzone-bar="false"></div>');
+
+            expect(utils.getOptionsFromAttrs($node[0]))
+                .to.deep.equal({ foo: true, bar: false });
+        });
+
+        it('should parse numbers', () => {
+            const $node = $('<div data-dropzone-max-files="10"></div>');
+
+            expect(utils.getOptionsFromAttrs($node[0]))
+                .to.deep.equal({ maxFiles: 10 });
+        });
+    });
+
+    describe('camelCasify()', () => {
+        it('should transform a hyphen serrated string to camel case', () => {
+            expect(utils.camelCaseIfy('foo-bar-baz')).to.equal('fooBarBaz');
+        });
+    });
+
+    describe('createFileNode()', () => {
+        const options = {
```

Comment:
This object should probably be recreated and assigned (so a `let` and not a `const`, ofc) inside a `beforeEach(...)`, in case the code accidentally modifies the object, in which case it could affect subsequent tests